transport/handler.go

```diff
@@ -22,7 +22,38 @@ package transport
 
 import "golang.org/x/net/context"
 
-// Handler handles a single transport-level request.
+// Mode is an enum of RPC types
+type Mode int
+
+const (
+	// Unknown RPC type
```

Comment:
Be consistent when referring to a concept - that will create and reinforce clarity with our users and developers.

Above you were calling this `RPCMode` and here `RPC type` - let's always call it `RPC Type` (casing matters).